website/docs/concepts/html/introduction.mdx

```diff
@@ -151,7 +151,41 @@ HTML code.
 At the moment the lints are mostly accessibility-related. If you have ideas for lints, please feel
 free to [chime in on this issue](https://github.com/yewstack/yew/issues/1334).
 
-## Special properties
+## Specifying attributes and properties
+
+Attributes are set on elements in the same way as in normal HTML:
+
+```rust
+let value = "something";
+html! { <div attribute={value} />
```

Comment:
This line seems to be missing a closing bracket. Shouldn't the website tests have been run on the PR to catch this?